notes/stories/film/film-scifi-1950s.st.txt

```diff
@@ -4074,6 +4074,49 @@ flying saucer [The Mysterians flew in a flying saucer.]
 giant robot
 
 
+movie: Terror from the Year 5000 (1958)
+=======================================
+
+:: Title
+Terror from the Year 5000
+
+:: Description
+Terror from the Year 5000 (a.k.a. Cage of Doom in the UK) is a 1958
+independently made American black-and-white science fiction film based
+(uncredited) on the short story "Bottle Baby" by print/TV/film writer Henry
+Slesar that was published in the science fiction magazine Fantastic (April
+1957).
+
+Synopsis: A nuclear physicist and his assistant build a machine that
+transports objects from the future to the present.
+
+:: Date
+1958
+
+:: References
+https://en.wikipedia.org/wiki/Terror_from_the_Year_5000
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+the dangers of unfettered scientific advancement [A message of the story was that Victor's reckless fast tracking of the use of a prototype time machine directly resulted in a murderous mutant monster being transported from the year 5000 AD to the present day.]
+time machine [A main novelty of the film is a large, immobile time machine that can transport people and objects from to and from the future.]
+fiancée and fiancée [Victor and Claire were engaged to be married.]
+father and daughter [Professor Erling and his adult daughter Claire were there throughout the story.]
+risk taking vs. playing it safe [There was the following ongoing conflict between Professor Erling and Victor: Victor was determined to plow ahead with testing out the time machine in spite of the risks while Erling urged they take a more cautious path involving additional testing and consulting with outside experts.]
+humanoid mutant [Victor inadvertently transported a murderous mutant from the year 5000 AD to the present.]
+existential risk to civilization [The people of the year 5000 AD were at risk of going extinct owing to the atmosphere being poisoned by radiation. The reason behind how the atmosphere had become irradiated was never spelled out.]
```

Comment:
Given the various hints and the cultural context of the 50s' America, I'd say we can read between the lines: 

https://www.themeontology.org/theme.php?name=nuclear%20holocaust

Reply:
The leap is more-or-less made in the plot summary of film's Wikipedia article. So sure, a **nuclear holocaust**, why not?